assets/sparksql/search.py

```
@@ -0,0 +1,19 @@
+
+spark = SparkSession \
+    .builder \
+    .appName("clp-bench") \
+    .master("spark://localhost:7077") \
+    .config("spark.sql.caseSensitive", True) \
+    .getOrCreate()
+
+parquetFile = spark.read.parquet(sys.argv[2])
+parquetFile.createOrReplaceTempView("parquetFile")
+
+results = spark.sql(f"SELECT * FROM parquetFile WHERE {sys.argv[1]};")
```

Ruff (0.12.2)

Possible SQL injection vector through string-based query construction

(S608)